docs/user/network/dashmate/index.rst

```diff
@@ -494,6 +513,166 @@ To disable logging to a file outside the container, reset the log path to ``null
 
    dashmate config set core.log.filePath null
 
+.. _dashmate-logs-platform:
+
+Configure Platform logs
+-----------------------
+
+For troubleshooting flexibility, dashmate provides independent log configuration for the Platform
+Gateway, Drive ABCI, and Tenderdash. Each service can be configured with the most helpful log level
+and output format.
+
+.. _dashmate-logs-platform-gateway:
+
+Gateway logs
```

Comment:
Gateway has two different log types:
- Service logs they go to stdout and we can control verbosity level `platform.gateway.log.level` (?)
- Access logs (requests) they are configured with `platform.gateway.log.accessLogs`.

When you add stdout access log then they are mixed together in stdout.

Reply:
Added info below